src/components/BorrowerProfile/DetailsTabs.vue

```diff
@@ -197,6 +197,26 @@ export default {
 		trusteeTabId() {
 			return `tab-panel-${this.name}-trustee`;
 		},
+		isSupporter() {
+			return this.loan.lentTo;
```

Comment:
prob want too use optional chaining on `loan` here.

Reply:
Same below

Reply:
@mcstover Below are checks looking at the value and not assigning it. It was my understanding that optional chaining is only needed when making an assignment to a const. Is that wrong?

Reply:
Yep, that's good. Please add your "[nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator)" operator so this returns `false` if it fails.